src/Config/Config.ts

```diff
@@ -19,7 +19,13 @@ const config = Object.freeze({
     port: env.SERVER_PORT || 3000,
     jwtKey: process.env.JWT_KEY || "nodefactory",
     jwtExpiry: process.env.JWT_EXPIRY || '24h',
-    rewardAmount: process.env.REWARD_AMOUNT || 5
+    rewardAmount: process.env.REWARD_AMOUNT || 5,
+    sendinblue: {
+        // eslint-disable-next-line max-len
+        apiKey: process.env.EMAIL_API_KEY || "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",
+        apiUrl: process.env.EMAIL_API_URL || "https://api.sendinblue.com/v3/smtp/email",
+        templateId: process.env.EMAIL_TEMPLATE_ID || "2",
```

Comment:
we will have confirmation email too so put the purpose as the key name i.e. downtimeTemplateId

Reply:
@morrigan I will make a few more changes with that in mind to support sending multiple types of mails.